crates/lib/src/executable.rs

```diff
@@ -225,21 +227,24 @@ impl<'executable> Executable<'executable, '_> {
     ///             .with_parameter("theta", 0, theta)
     ///             .with_parameter("theta", 1, theta * 2.0)
     ///             .execute_on_qvm().await.unwrap();
-    ///         let theta_register = readout_data
-    ///             .to_readout_map()
+    ///         let theta_register = result
+    ///             .result_data
+    ///             .to_register_map()
     ///             .expect("should fit readout map")
     ///             .get_register_matrix("theta")
     ///             .expect("should have theta")
     ///             .as_real()
     ///             .expect("should be real valued register")
-    ///             .to_owned()
+    ///             .to_owned();
```

Comment:
Consistency note: this has `to_owned()`, while the others use `clone()`.

Reply:
I went ahead and made them all use `to_owned()`